translation/en/RedirectedResources/assets/abdata/adv/scenario/c-9/00/0/zz_machineTranslation.txt

```diff
@@ -81,7 +81,7 @@
 「呼び捨てで良いってば。変な所で堅いのね、あんた」=“I don't mind if you call me that. You're a little uptight in a weird way, aren't you?”
 「ご、ごめん。でも呼び捨てか。んーっと……桜？」=“I'm sorry. I'm sorry, but you're calling me that. I'm sorry, I'm sorry... cherry blossoms?”
 「――っ！？」=“I'm sorry, I didn't mean to wake you.”
-「……」=“...”
+「……」=...
```

Comment:
These always seem to loose the quotes

Reply:
It pulls from translation.txt files so the dots must be that way in finished translations somewhere.